model/src/main/kotlin/AnalyzerResult.kt

```diff
@@ -47,7 +47,7 @@ data class AnalyzerResult(
         val packages: SortedSet<CuratedPackage>,
 
         /**
-         * The list of all errors.
+         * The lists of errors that occurred when analyzing the identified project.
```

Comment:
To make it even more clear, you could add something like: "This only contains error that occurred with the project itself. Errors related to dependencies of the project are embedded within the dependencies of the project's scopes."

Reply:
Done, with minor rewording.